src/Components/ServiceScene/LogListControls.tsx

```diff
@@ -63,6 +68,20 @@ export const LogListControls = ({
         tooltip={sortOrder === LogsSortOrder.Descending ? 'Newest logs first' : 'Oldest logs first'}
         size="lg"
       />
+      {wrapLogMessage !== undefined && onWrapLogMessageClick && (
+        <IconButton
+          name="wrap-text"
+          className={wrapLogMessage ? styles.controlButtonActive : styles.controlButton}
+          aria-pressed={wrapLogMessage}
+          onClick={() => onWrapLogMessageClick(!wrapLogMessage)}
```

Comment:
Am I trippin' is this backwards?
This looks wrapped to me, but the button is not highlighted and the text says 'wrap'
![Screenshot 2025-07-09 at 12 29 50 PM](https://github.com/user-attachments/assets/4ab52894-004d-46c1-94ee-605d51363c2e)
This looks unwrapped but the button is highlighted and the text say 'unwrap'
![Screenshot 2025-07-09 at 12 30 03 PM](https://github.com/user-attachments/assets/a8a7ea81-c394-4f92-9c8a-9c00c804b409)

Reply:
Yup it's backwards. Also I noticed that while we are using the same local storage key as logs panel/explore, the logs panel in logs drilldown saves the URL state, so updates in JSON don't update the logs panel unless the user comes into the app fresh.

Reply:
Fixed in https://github.com/grafana/logs-drilldown/pull/1388/commits/59c7231687e811ae6a093c961f0c4f63e95c0d8f

Url sync fixed in https://github.com/grafana/logs-drilldown/pull/1388/commits/f92d5e178e2574ee9a59879dd917831e4138eabb.

Also there was a bug where the visualization type was not url synced either (or working with JSON) which should be fixed now as well